frontend/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import ROUTES from '../constants/routers';
+import styles from './Footer.module.scss';
+import IconButton from '../ui-kit/components/IconButton';
+import { ICONS } from '../ui-kit/icons';
+
+const kvitkovoLinks = [
+  { route: ROUTES.about, text: 'Про нас' },
+  { route: ROUTES.contacts, text: 'Контакти' },
+  { route: ROUTES.partner, text: 'Стати партнером' },
+  { route: ROUTES.privacy, text: 'Політика конфіденційності' },
+];
+
+const clientLinks = [
+  { route: ROUTES.orderStatus, text: 'Статус замовлення' },
+  { route: ROUTES.delivery, text: 'Доставка та оплата' },
+  { route: ROUTES.about, text: 'Акції в Kvitkovo' },
+  { route: ROUTES.care, text: 'Догляд за квітами' },
+  { route: ROUTES.faq, text: 'Поширені запитання' },
+];
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className={styles.container}>
+        <section>
+          <h3 className={styles.title}>Kvitkovo</h3>
+          <ul>
+            {kvitkovoLinks.map((link, index) => (
+              <li key={index}>
+                <NavLink className={styles.links} to={link.route}>
+                  {link.text}
+                </NavLink>
+              </li>
+            ))}
+          </ul>
+        </section>
+        <section>
+          <h3 className={styles.title}>Клієнтам</h3>
+          <ul>
+            {clientLinks.map((link, index) => (
+              <li key={index}>
```

Comment:
Index is used in the most extreme case, it is better to take route, they are not repeated